ArmoryQt.py

```diff
@@ -2959,9 +2959,9 @@ def createCombinedLedger(self, resetMainLedger=False):
       if bdmState == BDM_BLOCKCHAIN_READY:
          for wltID in self.wltIDList:
             wlt = self.walletMap[wltID]
-            totalFunds += wlt.getBalance('Total')
-            spendFunds += wlt.getBalance('Spendable')
-            unconfFunds += wlt.getBalance('Unconfirmed')
+            totalFunds += int(wlt.getBalance('Total'))
```

Comment:
Why did you have to cast these to ints? These methods should be returning uint64_t which swig will turn into a natural python int